// react-reconciler/cjs/react-reconciler-constants.development.js
'use strict';

require('../../../_virtual/_commonjsHelpers.js');
var reactReconcilerConstants_development = require('../../../_virtual/react-reconciler-constants.development.js');

/**
 * @license React
 * react-reconciler-constants.development.js
 *
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

var hasRequiredReactReconcilerConstants_development;

function requireReactReconcilerConstants_development () {
	if (hasRequiredReactReconcilerConstants_development) return reactReconcilerConstants_development.__exports;
	hasRequiredReactReconcilerConstants_development = 1;
	"use strict";
	"production" !== process.env.NODE_ENV &&
	  ((reactReconcilerConstants_development.__exports.ConcurrentRoot = 1),
	  (reactReconcilerConstants_development.__exports.ContinuousEventPriority = 8),
	  (reactReconcilerConstants_development.__exports.DefaultEventPriority = 32),
	  (reactReconcilerConstants_development.__exports.DiscreteEventPriority = 2),
	  (reactReconcilerConstants_development.__exports.IdleEventPriority = 268435456),
	  (reactReconcilerConstants_development.__exports.LegacyRoot = 0),
	  (reactReconcilerConstants_development.__exports.NoEventPriority = 0));
	return reactReconcilerConstants_development.__exports;
}

exports.__require = requireReactReconcilerConstants_development;
