// react-reconciler/cjs/react-reconciler-constants.production.js
'use strict';

require('../../../_virtual/_commonjsHelpers.js');
var reactReconcilerConstants_production = require('../../../_virtual/react-reconciler-constants.production.js');

/**
 * @license React
 * react-reconciler-constants.production.js
 *
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

var hasRequiredReactReconcilerConstants_production;

function requireReactReconcilerConstants_production () {
	if (hasRequiredReactReconcilerConstants_production) return reactReconcilerConstants_production.__exports;
	hasRequiredReactReconcilerConstants_production = 1;
	"use strict";
	reactReconcilerConstants_production.__exports.ConcurrentRoot = 1;
	reactReconcilerConstants_production.__exports.ContinuousEventPriority = 8;
	reactReconcilerConstants_production.__exports.DefaultEventPriority = 32;
	reactReconcilerConstants_production.__exports.DiscreteEventPriority = 2;
	reactReconcilerConstants_production.__exports.IdleEventPriority = 268435456;
	reactReconcilerConstants_production.__exports.LegacyRoot = 0;
	reactReconcilerConstants_production.__exports.NoEventPriority = 0;
	return reactReconcilerConstants_production.__exports;
}

exports.__require = requireReactReconcilerConstants_production;
